<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN"> 
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Foundations of Information Science : stdin : [Fis] ON MOLECULAR BIONETWORKS (II)</title>

<script language="javascript" src="../scripts/sniffer.js"></script>
<script language="javascript1.2" src="../scripts/custom.js"></script>
<script language="javascript1.2" src="../scripts/style.js"></script>

</head>

<body topmargin="8" link="#647286" vlink="#647286" alink="#C7D3DA">
<p><a href="../default.htm"><img src="../images/logo.gif" align="top" border="0" WIDTH="474" HEIGHT="33"></a></p><br>

<table width="800"><td><br>

<u><h3>[Fis] ON MOLECULAR BIONETWORKS (II)</h3></u>
<div class="head">
<h1>[Fis] ON MOLECULAR BIONETWORKS (II)</h1>
<!-- received="Mon Nov 21 08:04:48 2005" -->
<!-- isoreceived="20051121070448" -->
<!-- sent="Sun, 20 Nov 2005 20:06:50 -0500" -->
<!-- isosent="20051121010650" -->
<!-- name="Jerry LR Chandler" -->
<!-- email="Jerry.LR.Chandler@Cox.net" -->
<!-- subject="[Fis] ON MOLECULAR BIONETWORKS (II)" -->
<!-- id="21379E70-1F7D-410D-A18D-E614562EEAAF@Cox.net" -->
<!-- charset="WINDOWS-1252" -->
<!-- inreplyto="200511081019.jA8AJ8dT030125&#64;relay.unizar.es" -->
<!-- expires="-1" -->
<map id="navbar" name="navbar">
<ul class="links">
<li>
<dfn>This message</dfn>:
[ <a href="#start" name="options1" id="options1" tabindex="1">Message body</a> ]
 [ <a href="#options2">More options</a> ]
</li>
<li>
<dfn>Related messages</dfn>:
<!-- unext="start" -->
[ <a href="2236.html" title="Loet Leydesdorff: &quot;[Fis] Journal Citation Reports of _Science Citation Index_ and _SocialScience Citation Index_ combined&quot;">Next message</a> ]
[ <a href="2234.html" title="Aleks Jakulin: &quot;Re: FW: [Fis] The Molecule as Text&quot;">Previous message</a> ]
<!-- unextthread="start" -->
 [ <a href="2250.html" title="Jerry LR Chandler: &quot;[Fis] Re: ON MOLECULAR BIONETWORKS (III)&quot;">Next in thread</a> ]
 [ <a href="#replies">Replies</a> ]
<!-- ureply="end" -->
</li>
</ul>
</map>
</div>
<!-- body="start" -->
<div class="mail">
<address class="headers">
<span id="from">
<dfn>From</dfn>: Jerry LR Chandler &lt;<a href="mailto:Jerry.LR.Chandler&#64;Cox.net?Subject=Re:%20[Fis]%20ON%20MOLECULAR%20BIONETWORKS%20(II)">Jerry.LR.Chandler@Cox.net</a>&gt;
</span><br />
<span id="date"><dfn>Date</dfn>: Mon 21 Nov 2005 - 02:06:50 CET</span><br />
</address>
<p>
Dear Colleagues:
<br />
<p>I have just returned from an stimulating meeting on the philosopher /  
<br />
scientist  De Chardin.  His global perspectives are a truly  
<br />
remarkable synthesis - even after one sorts out the &quot;noise&quot; factors.   
<br />
It makes the challenges of Bionetworks seem more approachable.  :-)
<br />
<p>Since several responses have commented either directly or indirectly  
<br />
on logic, this message will seek to explore some facets of relations  
<br />
between chemical logic and other logics.  Thus, I will address the  
<br />
first question in my list:
<br />
<p><em class="quotelev1">&gt; 1. From an informational perspective what would constitute a  
</em><br />
<em class="quotelev1">&gt; definitive theory of the information content of molecular bionetworks?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<p>First, two historical comments that are not part of the general  
<br />
scientific folklore or narrative generation (that Stan is so fond  
<br />
of. :-) )
<br />
<p>About 1875, after Kirchoff's laws had penetrated the scientific and  
<br />
mathematical communities, two deep results on chemical logic came  
<br />
into existence.
<br />
<p>One conclusion was that the networks of Kirchoff could not be used to  
<br />
describe chemical structures.  Nonetheless, the &quot;drawings&quot; of  
<br />
Kirchoff's networks and the &quot;drawings&quot; of chemical structures look  
<br />
very very similar.  If we call such drawings by the formal  
<br />
mathematical name of &quot;graphs&quot;, then the Graphs of Kirchoff morphed  
<br />
into groups and became closely associated with differential equations  
<br />
and factoring of products (polynominals), linear algebra.  Thus,  
<br />
chemical theory separated from mathematical theory.
<br />
<p>Secondly, the optical isomer problem of tartaric acid initiated the  
<br />
general problem of isomers - substances of the same molecular formula  
<br />
but different physical - chemical properties
<br />
<p>&nbsp;&nbsp;So, what is the logic of a chemical structure?  This is a crucial  
<br />
question.  If we can not answer this question, how are we to approach  
<br />
the logic of bionetworks? of genetics? simulation?   Is an  
<br />
alternative grounding (not chemical structures) for these logics   
<br />
conceivable?
<br />
<p>It is important to note that the physical community generally  
<br />
presupposes that &quot;quantum mechanics&quot;  or &quot;symmetry&quot;  or  
<br />
&quot;thermodynamics&quot;  provides the logic to solve these problems.  I  
<br />
invite individuals to address the relations between physical theories  
<br />
and these two questions - it would be a healthy opening for further  
<br />
discussions.
<br />
<p>One of the problems with the logic of chemistry is the relation  
<br />
between principle of the conservation of matter and the nature of  
<br />
arithmetic operations.  The conservation of matter demands that each  
<br />
chemical particle is preserved in its individuality.
<br />
<p>Example: The atomic number of Hydrogen is one and of Carbon is six.
<br />
Thus, we speak of C6 H6 (Benzene)  as 6*6 and 6 * 1 as 36 + 6 = 42.
<br />
But 42 is neither the molecular formula nor molecular weight of benzene.
<br />
Thus, a significant logical separation of the nature of numbers of  
<br />
chemistry from the nature of numbers of mathematics exists.  This  
<br />
distinction casts a long shadow over the concept of bionetworks.
<br />
<p>Ted states:
<br />
<p>&quot;In the local context, all you have to do is invent a new science  
<br />
that sheds new light on phenomenon that are seen locally, namely in  
<br />
the molecular domain.&quot;
<br />
<p>In an overarching view, I agree with Ted that &quot;all you have to do is  
<br />
invent a new science&quot;.  But, in the case of chemistry, it was also  
<br />
necessary to invent a new mathematics.  This new mathematics is the  
<br />
basis of  bionetworks.  In addition one has to invent a method of  
<br />
scaling because arithmetic operations do not work on atomic numbers.
<br />
<p>Ted continues:
<br />
<p>&quot;In the FIS context, you have this seeming impossibility squared.  
<br />
That's because you'll be using this domain to get leverage on the  
<br />
larger problem of an information science that applies at all domains.  
<br />
And more importantly, we'd understand what flows among domains and how.&quot;
<br />
<p>I agree with the sentiment but not the impossibility of doing it.
<br />
I think we must seek deeper abstract methods of understanding the  
<br />
sources of communication.  All to often, we tend to fall into  
<br />
scientific or philosophical jargon and reduce the concept of  
<br />
communication to the concept of probability and then to the concept  
<br />
of &quot;transmission&quot; in a linear symbol sequence, and then into  
<br />
simulation of symbol behavior.  Rather than converging to  
<br />
calculations, we must think in more abstract views that have the  
<br />
potential to link to calculations at some latter logical step.
<br />
<p>This approach of immediate convergence is &quot;going the wrong direction&quot;  
<br />
in my view.  We ought to first diverge into wider philosophical  
<br />
territory that can become a basis for latter calculations.  Pedro has  
<br />
suggested some possible avenues that deserve attention when he writes:
<br />
<p>In our times, I would say that the &quot;bioinformational&quot; themes have  
<br />
definitely gone beyond the unitary schemes proposed after the  
<br />
adoption of the information metaphor (codes, signals, translation,  
<br />
transduction, etc.), for instance, the case of the so called Central  
<br />
Dogma of molecular biology.
<br />
<p>The challenge is to go beyond the &quot;genetic code.&quot;  The code is now  
<br />
more than 40 years old.  It itself was strongly influenced by  
<br />
Shannon / Weaver/  Weiner, etc.
<br />
<p>Yes, molecular biology has progressed dramatically over the past 45  
<br />
years - but almost exclusively on the basis of empirical evidence,  
<br />
not new methods of calculations.  The major exception to this wide  
<br />
claim are the raw combinatorial calculations for comparing sequences  
<br />
of various polymers.  By comparing sequences (proteins, DNA) we have  
<br />
illuminated the pathways of evolution of species.  Remarkable  
<br />
progress, heavily dependent on statistics but only lightly touched by  
<br />
information theory.
<br />
<p>To return to the logical issues of communication, classical logic is  
<br />
so &quot;hung up&quot; on the grammar of predication that formal analogy with  
<br />
chemical bionetworks appears hopeless.  Bio-logic appears to me to  
<br />
vastly deeper and richer than either propositional logic or first  
<br />
order logic.  Pedro appears to agree with this statement when he writes:
<br />
<p>&quot;So the ‘what’ of the functional clause should be accompanied by  
<br />
circumstances such as ‘how fast’, ‘where’, ‘which way’, ‘with whom’,  
<br />
‘when’, and ‘how long’.&quot;
<br />
<p>This comment of Pedro recalls Robert Rosen's text in Anticipatory  
<br />
Systems on the role of interrogatories in general.  The critical  
<br />
point is that chemical relations among molecules are vastly richer  
<br />
than the notion of a variable.  The conformation of a protein  
<br />
molecule can occur in many different forms and functional states.   
<br />
So, the concept of a molecule should not be confused with the concept  
<br />
of a variable.  The pathways of empirical evidence are open to  
<br />
exploration but the methods to express the formal logic are  
<br />
deficient.  In a practical sense, if a protein of 10,000 atoms  
<br />
interacts with three substrates, each of 100 atoms, then how can we  
<br />
express the logic?  Empirically, I think that chemists use a private  
<br />
version of Aristotle's ten categories.  In some sense or another, if  
<br />
the protein and the three substrates come from the same organism, the  
<br />
molecules &quot;know each other&quot;. By this I mean that the mutual source of  
<br />
co-emergence of the bionetwork has adapted the structures so that  
<br />
each contributes to the reproduction in a meaning way without  
<br />
untoward contempt for one another. But modern philosophers have  
<br />
rejected Aristotle's categories.  (See for example, Logical  
<br />
Properties by Colin McGinn.)
<br />
<p>Thus, I sense a deep logical disconnect between the philosophical  
<br />
approach to logic, the mathematical approach to logic and the  
<br />
chemist's approach to logic.  From an evolutionary perspective,  
<br />
chemical logic must anticipate both propositional logic and model  /  
<br />
mathematical logic. (I believe that this is an historical fact that  
<br />
some may find distasteful.  Other viewpoints on this critical matter  
<br />
are desirable.)
<br />
<p>Ted continue:
<br />
<p>&quot;Toward this end, I'd like to put an early suggestion for limiting  
<br />
the &quot;bio&quot; semiotic approach a bit. Several smart people here have  
<br />
noted the internal/external problem of meaning and intent when you  
<br />
scale biosemiotics down to elements that apparently cannot reason. So  
<br />
I am very skeptical of Peircian mechanics at the biological level.&quot;
<br />
<p>In recent discussions on the Pierce listserve, it was concluded that  
<br />
Piercian semiotics is infinite.  Roughly,the reasoning was as  
<br />
follows:  If sign B can replace sign A, then sign C can replace sign  
<br />
B, and so forth.  Simple transitivity of sign replacement.    (This  
<br />
view is foreign to most discussions of Piercian semiotics I have  
<br />
seen!)   So, I must ask Ted for two clarifications.
<br />
The internal / external issues have physical and cellular  
<br />
counterparts.  What is the reference?
<br />
The critical role of distinguishing logically among internal /  
<br />
external issues drives me to the concept of organization levels and  
<br />
the separation of categories by the choice of classes (categories)  
<br />
and relations.
<br />
<p>How are we to interpret the concept of genetic reasoning?  By genetic  
<br />
reasoning, I mean the choices of an organism to &quot;turn-on&quot; some genes  
<br />
and &quot;turn -off&quot; others in response to the external environment?    
<br />
&quot;Choices&quot; in this context  can be used in the informational sense.
<br />
<p><p>Further discussions of logics are needed; I definitely concur with  
<br />
Ted on this issue.  The challenge is to find ways to communicate  
<br />
abstractions among individuals with widely different knowledge bases  
<br />
and experiences.   I have not been able to extend the Barwise  
<br />
approach to chemical systems - has anyone any ideas on how this might  
<br />
be approached?
<br />
<p>Ted suggests:
<br />
<p>I suggest we consider artistic and humanity-centric &quot;logics&quot; also, as  
<br />
we hunt for tools, and be open to a scope that includes internal  
<br />
conceptual mechanics: desires, intuitions, emotions, creativity.
<br />
<p>This sentence reminds me of the seminal works of  (Howard?) Simon and  
<br />
of Christopher Alexander on the concept of design.  We want to design  
<br />
what?  A new symbol system?  A new perspective logic?  A new view of  
<br />
relations?  A better approach to human communication?  What would be  
<br />
an appropriate source of abstractions for such designs?  These are  
<br />
daunting questions!
<br />
<p>Thus, I  leave these questions as open to individual  
<br />
interpretations.  Perhaps the philosophers can open up these  
<br />
questions?   In what sense can the concept of interrogatives be  
<br />
leveraged to openings?
<br />
<p>Several issues are left un-addressed, including Kevin's interesting  
<br />
posts and Koichiro's last post in the previous session.  More in my  
<br />
next post.
<br />
<p>Cheers
<br />
<p>Jerry
<br />
<p><p>On Nov 8, 2005, at 5:19 AM, fis-request&#64;listas.unizar.es wrote:
<br />
<p><em class="quotelev1">&gt; Send fis mailing list submissions to
</em><br />
<em class="quotelev1">&gt;     fis&#64;listas.unizar.es
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; To subscribe or unsubscribe via the World Wide Web, visit
</em><br />
<em class="quotelev1">&gt;     <a href="../../webmail.unizar.es/mailman/listinfo/fis">http://webmail.unizar.es/mailman/listinfo/fis</a>
</em><br />
<em class="quotelev1">&gt; or, via email, send a message with subject or body 'help' to
</em><br />
<em class="quotelev1">&gt;     fis-request&#64;listas.unizar.es
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; You can reach the person managing the list at
</em><br />
<em class="quotelev1">&gt;     fis-owner&#64;listas.unizar.es
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; When replying, please edit your Subject line so it is more specific
</em><br />
<em class="quotelev1">&gt; than &quot;Re: Contents of fis digest...&quot;
</em><br />
<em class="quotelev1">&gt; Today's Topics:
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;    1. ON MOLECULAR BIONETWORKS (Jerry LR Chandler)
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; From: Jerry LR Chandler &lt;Jerry.LR.Chandler&#64;Cox.net&gt; (by way of  
</em><br />
<em class="quotelev1">&gt; Pedro Marijuan &lt;marijuan&#64;unizar.es&gt;)
</em><br />
<em class="quotelev1">&gt; Date: November 8, 2005 5:20:33 AM EST
</em><br />
<em class="quotelev1">&gt; To: fis&#64;listas.unizar.es
</em><br />
<em class="quotelev1">&gt; Subject: [Fis] ON MOLECULAR BIONETWORKS
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 9th FIS Discussion Session:
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; ON MOLECULAR BIONETWORKS
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Jerry L.R. Chandler
</em><br />
<em class="quotelev1">&gt; Research Professor
</em><br />
<em class="quotelev1">&gt; Krasnow Institute for Advanced Study
</em><br />
<em class="quotelev1">&gt; George Mason University
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Kevin G. Kirby
</em><br />
<em class="quotelev1">&gt; Department of Computer Science,
</em><br />
<em class="quotelev1">&gt; Northern Kentucky University
</em><br />
<em class="quotelev1">&gt; Highland Heights, Kentucky
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Introductory Remarks 1:
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 12 Questions about relations between molecular bionetworks and  
</em><br />
<em class="quotelev1">&gt; information theory
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; (by Jerry L.R. Chandler)
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; I agreed to write a brief introduction to the subject, knowing that  
</em><br />
<em class="quotelev1">&gt; I could not do justice to such a complex topic in such a short  
</em><br />
<em class="quotelev1">&gt; time.  As little is known about the subject, I chose to introduce  
</em><br />
<em class="quotelev1">&gt; topics and to pose questions about potential relations.  The  
</em><br />
<em class="quotelev1">&gt; questions are significant but not ordered by importance or  
</em><br />
<em class="quotelev1">&gt; priority. Hopefully, participants will contribute to exploring the  
</em><br />
<em class="quotelev1">&gt; meanings of the questions and addressing conceivable approaches to  
</em><br />
<em class="quotelev1">&gt; answers.  Perhaps one objective could be to discuss the relative  
</em><br />
<em class="quotelev1">&gt; importance of the 12 questions.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;  The concept of molecular bionetworks is a modern concept.  The  
</em><br />
<em class="quotelev1">&gt; development of this concept will continue into the indefinite  
</em><br />
<em class="quotelev1">&gt; future.  That is to say, the concept is, at present, well defined  
</em><br />
<em class="quotelev1">&gt; but in a primitive or early stage of development.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 1. From an informational perspective what would constitute a  
</em><br />
<em class="quotelev1">&gt; definitive theory of the information content of molecular bionetworks?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Historically, in the 1840’s, Kirchoff’s theory of flow in  
</em><br />
<em class="quotelev1">&gt; electrical networks grounds the metaphor of the conceptual  
</em><br />
<em class="quotelev1">&gt; framework for molecular bionetworks.  However, the relatively  
</em><br />
<em class="quotelev1">&gt; simplicity of the flow of electrons in comparison to the flow of  
</em><br />
<em class="quotelev1">&gt; life restricts the analogy severely.  Nevertheless, biomolecules  
</em><br />
<em class="quotelev1">&gt; are composed from electrical particles and certain electron flow  
</em><br />
<em class="quotelev1">&gt; patterns are intrinsic to metabolic networks in living systems.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 2. From the perspective of information, how is it possible that  
</em><br />
<em class="quotelev1">&gt; bionetworks construct the informed flows of electrical current flow  
</em><br />
<em class="quotelev1">&gt; and how is it related metabolic flows?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Historically, the investigation of empirical basis of molecular  
</em><br />
<em class="quotelev1">&gt; bionetworks (metabolism) started shortly after Pastuer’s pioneering  
</em><br />
<em class="quotelev1">&gt; experiments on the causes of fermentation (1870’s).  At that time,  
</em><br />
<em class="quotelev1">&gt; the fermentation of grape juice was difficult to control but of  
</em><br />
<em class="quotelev1">&gt; great economic importance.  (Bad wine sells cheaply!)  Quantitative  
</em><br />
<em class="quotelev1">&gt; analysis of yeast fermentations showed that each molecule of  
</em><br />
<em class="quotelev1">&gt; glucose generated two molecules of the 2-carbon alcohol and two  
</em><br />
<em class="quotelev1">&gt; molecules of carbon dioxide.  The yeast cell informed the specific  
</em><br />
<em class="quotelev1">&gt; destruction of the sugar.  (If the 6-carbon sugar was merely  
</em><br />
<em class="quotelev1">&gt; burned, then it produced exactly six molecules of carbon dioxide.)   
</em><br />
<em class="quotelev1">&gt; Thus, the thermodynamic question of why the yeast cell did not  
</em><br />
<em class="quotelev1">&gt; completely burn up the 6- carbon sugar entirely into carbon dioxide  
</em><br />
<em class="quotelev1">&gt; arose.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 3. From an informational perspective, what does this incomplete  
</em><br />
<em class="quotelev1">&gt; fermentation process of glucose suggest about the role of  
</em><br />
<em class="quotelev1">&gt; thermodynamics in living processes?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Continuing with the historical roots of biomolecular networks,  
</em><br />
<em class="quotelev1">&gt; early in the 20 th Century, the exact chemical pathway between  
</em><br />
<em class="quotelev1">&gt; glucose and alcohol and CO2 was decoded.  The code of the cell for  
</em><br />
<em class="quotelev1">&gt; fermentation was that each chemical reaction was catalysed by an  
</em><br />
<em class="quotelev1">&gt; informed agents, termed  “zwishenferments”.  These agents are now  
</em><br />
<em class="quotelev1">&gt; known as enzymes. Thus, the question of how a cell knows how to  
</em><br />
<em class="quotelev1">&gt; ferment sugar was regressed to a deeper informational question:   
</em><br />
<em class="quotelev1">&gt; How does an enzyme know how to inform a chemical change?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 4. From an informational perspective, what is the nature of the  
</em><br />
<em class="quotelev1">&gt; code that an enzyme contains such that it conducts an informed  
</em><br />
<em class="quotelev1">&gt; catalytic process?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Continuing with the historical perspective, by the 1950s, it was  
</em><br />
<em class="quotelev1">&gt; known that biological information was carried in chemical  
</em><br />
<em class="quotelev1">&gt; sequences, enzymes were composed from amino acid sequences and that  
</em><br />
<em class="quotelev1">&gt; nucleic acids were composed from nucleotides.  In the 1960’s, the  
</em><br />
<em class="quotelev1">&gt; convergence of experimental work in genetics, biochemistry, x-ray  
</em><br />
<em class="quotelev1">&gt; crystallography, nutrition and Shannon’s coding theory, generated  
</em><br />
<em class="quotelev1">&gt; the hypothesis that biological information is encoded in chemical  
</em><br />
<em class="quotelev1">&gt; structures.  This hypothesis remains intact today.  During the  
</em><br />
<em class="quotelev1">&gt; period from 1970 to the present, methods for determining the  
</em><br />
<em class="quotelev1">&gt; chemical sequences of biomolecules were developed and automated.   
</em><br />
<em class="quotelev1">&gt; Today, sequencing of microgram quantities of biomolecules is  
</em><br />
<em class="quotelev1">&gt; routine.  The complete DNA sequences of hundreds of biological  
</em><br />
<em class="quotelev1">&gt; species are known.  The sequences of tens of thousands of proteins  
</em><br />
<em class="quotelev1">&gt; are known.  The sizes of proteins, the so-to-speak “cybernetic  
</em><br />
<em class="quotelev1">&gt; agents,” are often hundreds as time as large as the small molecules  
</em><br />
<em class="quotelev1">&gt; that they control.  Interpolation between protein sequences and DNA  
</em><br />
<em class="quotelev1">&gt; sequences are routinely conducted via computer programs.  The  
</em><br />
<em class="quotelev1">&gt; relation between the information in DNA or in proteins and the  
</em><br />
<em class="quotelev1">&gt; substrates for catalytic reactions is deeply mysterious.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 5. From an informational perspective, given solely the DNA and  
</em><br />
<em class="quotelev1">&gt; protein sequences of an organism, what can we say about the  
</em><br />
<em class="quotelev1">&gt; biological functioning of the organism?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; The classification of chemical structures has been given a new set  
</em><br />
<em class="quotelev1">&gt; of names in the past decade.  Genomics becomes the study of the  
</em><br />
<em class="quotelev1">&gt; genome, the DNA sequences and properties; protonomic becomes the  
</em><br />
<em class="quotelev1">&gt; study of protein structures and properties, metabolonomics becomes  
</em><br />
<em class="quotelev1">&gt; the study of “small” molecules where the sequences of atoms are  
</em><br />
<em class="quotelev1">&gt; relatively short.  (The use of the Greek root, “nomos”, as in law  
</em><br />
<em class="quotelev1">&gt; (for example, in autonomy), is somewhat misleading in this context.)
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Biomolecular networks consist of all the molecules in a cell.  One  
</em><br />
<em class="quotelev1">&gt; crucial feature of such networks is the capacity to generate  
</em><br />
<em class="quotelev1">&gt; biological functions.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 6. From an informational perspective, what is the nature of the  
</em><br />
<em class="quotelev1">&gt; information content of a collection of molecules such that they  
</em><br />
<em class="quotelev1">&gt; generate biological function?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; A second crucial feature of such networks is the capacity, in an  
</em><br />
<em class="quotelev1">&gt; appropriate ecosystem, to reproduce themselves with exactly the  
</em><br />
<em class="quotelev1">&gt; same chemical structures.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 7. From an informational perspective, what is the nature of the  
</em><br />
<em class="quotelev1">&gt; relational interactions between the interior and the exterior of an  
</em><br />
<em class="quotelev1">&gt; organism such that the information is reproduced?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 8. From an informational perspective, is the reproduction of a cell  
</em><br />
<em class="quotelev1">&gt; a mathematical calculation?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Biomolecular networks function in time.  In humans, the generation  
</em><br />
<em class="quotelev1">&gt; of temporal networks of the heart, the brain, the menses, and so  
</em><br />
<em class="quotelev1">&gt; forth are intricate flows of orderly information that manifest  
</em><br />
<em class="quotelev1">&gt; themselves in regular and chaotic rhythms.  In simple unicellular  
</em><br />
<em class="quotelev1">&gt; organisms, the growth rate and cellular division can follow strict  
</em><br />
<em class="quotelev1">&gt; rhythms.  Such rhythms can be modulated, altered, extended or  
</em><br />
<em class="quotelev1">&gt; stopped by chemical agents that are foreign to the bionetwork.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 9. From an informational perspective, what are the relations  
</em><br />
<em class="quotelev1">&gt; between chemical structures and temporal rhythms?  What is the  
</em><br />
<em class="quotelev1">&gt; information content of a chemical rhythm?
</em><br />
<em class="quotelev1">&gt; In what sense are the rhythms of temporal chemical relations the  
</em><br />
<em class="quotelev1">&gt; source of biological information?   What is the information content  
</em><br />
<em class="quotelev1">&gt; of a chemical rhythm?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Early in the history of biology, organisms were classified based on  
</em><br />
<em class="quotelev1">&gt; categories, more or less following Aristotelian conceptualization  
</em><br />
<em class="quotelev1">&gt; of Phyphrian trees, choices of potential properties.  The ten  
</em><br />
<em class="quotelev1">&gt; Aristotelian categories, the ten highest genera are listed as:  
</em><br />
<em class="quotelev1">&gt; substance, quantity, quality, relation, place, time, position,  
</em><br />
<em class="quotelev1">&gt; state, and action, being acted on.  Biological function is related  
</em><br />
<em class="quotelev1">&gt; to these categories.  I presume that a property of the molecular  
</em><br />
<em class="quotelev1">&gt; bionetwork encodes and decodes information concerning these genera.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 10. From modern information theory, how can we improve the  
</em><br />
<em class="quotelev1">&gt; Aristotelian categorization of properties of organisms, based on  
</em><br />
<em class="quotelev1">&gt; the qualitative and quantitative attributes of the biomolecular  
</em><br />
<em class="quotelev1">&gt; networks?  How should the concept of information be conceptualized  
</em><br />
<em class="quotelev1">&gt; such that it will illuminate communication about molecular  
</em><br />
<em class="quotelev1">&gt; bionetworks or biological function?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Each molecular bionetwork is specific to the particular species in  
</em><br />
<em class="quotelev1">&gt; which it exists.  However, the similarity of bionetworks roughly  
</em><br />
<em class="quotelev1">&gt; parallels the separation of species in the ‘tree of life.”   
</em><br />
<em class="quotelev1">&gt; Virtually all self-reproducing species include a common set of  
</em><br />
<em class="quotelev1">&gt; small molecules, amino acids, fats, sugars, vitamins, and  
</em><br />
<em class="quotelev1">&gt; minerals.  The degree of similarity of sequences of proteins, RNA  
</em><br />
<em class="quotelev1">&gt; and DNA vary with the “tree of life” separation.  Indeed,  
</em><br />
<em class="quotelev1">&gt; classification of species, genera and families are now based on  
</em><br />
<em class="quotelev1">&gt; calculation of sequence similarity.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 11. From an informational perspective, what is the significance of  
</em><br />
<em class="quotelev1">&gt; the sequence similarities for the theory of biological  
</em><br />
<em class="quotelev1">&gt; information?  For the concepts of encoding and decoding of  
</em><br />
<em class="quotelev1">&gt; molecular bionetworks?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; One approach to the partial analysis of the meaning of the genetic  
</em><br />
<em class="quotelev1">&gt; code was developed from Shannon’s theory by Dr. Thomas Schneider.
</em><br />
<em class="quotelev1">&gt;   <a href="../../www.lecb.ncifcrf.gov/~toms/default.htm">http://www.lecb.ncifcrf.gov/~toms/</a>
</em><br />
<em class="quotelev1">&gt;  The theory is specific for one aspect of the molecular bionetwork,  
</em><br />
<em class="quotelev1">&gt; namely binding of proteins to DNA.  His website provides a detailed  
</em><br />
<em class="quotelev1">&gt; account of Schneider’s philosophy.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; 12. From an informational perspective, the encoding processes for  
</em><br />
<em class="quotelev1">&gt; DNA – protein relationships are somehow related to Shannon’s  
</em><br />
<em class="quotelev1">&gt; theory.  What are the relations between this sort of encoding and  
</em><br />
<em class="quotelev1">&gt; other metabolic encoding?  In particular, can we imagine a  
</em><br />
<em class="quotelev1">&gt; catalytic – type of encoding that parallels the genetic encoding?
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;   
</em><br />
<em class="quotelev1">&gt; ---------------------------------------------------------------------- 
</em><br />
<em class="quotelev1">&gt; ------------------------------------------------------
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; Introductory Remarks 2:
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; The Natural Computation Perspective
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; (by Kevin Kirby)
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; If anything can be called a &quot;science of the artificial&quot; it is  
</em><br />
<em class="quotelev1">&gt; Computer Science, or so one might think.  In fact, such thinking is  
</em><br />
<em class="quotelev1">&gt; incorrect. And it is precisely in the field of biology --  with  
</em><br />
<em class="quotelev1">&gt; molecular biology as the best exemplar-- where a clearer  
</em><br />
<em class="quotelev1">&gt; understanding of Natural Computation will be so valuable.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; The initial encounter between computing and nature happens in the  
</em><br />
<em class="quotelev1">&gt; computational modeling of natural systems.  For example, we now see  
</em><br />
<em class="quotelev1">&gt; many object-oriented models of ecosystems. More deeply into  
</em><br />
<em class="quotelev1">&gt; computer science, we find questions such as, for example, whether a  
</em><br />
<em class="quotelev1">&gt; natural language has such-and-such a computational property (e.g.  
</em><br />
<em class="quotelev1">&gt; whether Dutch is a context-free language).  But beyond providing a  
</em><br />
<em class="quotelev1">&gt; way to talk about models of nature, some have viewed computer  
</em><br />
<em class="quotelev1">&gt; science as saying something about nature itself, constraining it in  
</em><br />
<em class="quotelev1">&gt; some way. A very strong form of the Church-Turing thesis can be  
</em><br />
<em class="quotelev1">&gt; taken as a physical law. And the theory of quantum information has  
</em><br />
<em class="quotelev1">&gt; made the &quot;it from bit&quot; slogan quite precise. It is in these latter  
</em><br />
<em class="quotelev1">&gt; senses that a true “natural computer science” begins to take shape.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; One way to further this approach is to turn to biological systems,  
</em><br />
<em class="quotelev1">&gt; and take a look at the simulation relation. How do we say a  
</em><br />
<em class="quotelev1">&gt; biological system computes X? Well, we see if there is a dynamics- 
</em><br />
<em class="quotelev1">&gt; preserving mapping between inputs and states of the biological  
</em><br />
<em class="quotelev1">&gt; system and a given formal system for X. This relation is usually  
</em><br />
<em class="quotelev1">&gt; written as a commutative diagram. The simulation relation is  
</em><br />
<em class="quotelev1">&gt; central in automata theory and was recast into a category-theoretic  
</em><br />
<em class="quotelev1">&gt; framework by Arbib and Goguen (taking different approaches).   But  
</em><br />
<em class="quotelev1">&gt; the notion of a mapping between a biological system and a formal  
</em><br />
<em class="quotelev1">&gt; system, seems to be, at first glance, a category mistake!  As soon  
</em><br />
<em class="quotelev1">&gt; as one identifies a fragment of nature as a system, one has locked  
</em><br />
<em class="quotelev1">&gt; in some set of states, and it is hard to separate the true  
</em><br />
<em class="quotelev1">&gt; computational power of a living system from what accrues merely to  
</em><br />
<em class="quotelev1">&gt; our conventional state assignment. This is taken up nicely by the  
</em><br />
<em class="quotelev1">&gt; philosopher David Chalmers in a response to a very strong statement  
</em><br />
<em class="quotelev1">&gt; at the conventionality end by Hilary Putnam. (One could see this as  
</em><br />
<em class="quotelev1">&gt; a recasting of the debate in Plato's Cratylus in computational terms!)
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; This tension between the formal and the material seems to lie at  
</em><br />
<em class="quotelev1">&gt; the heart of the field of natural computing. The work of Michael  
</em><br />
<em class="quotelev1">&gt; Conrad emphasized the special role of biological material to  
</em><br />
<em class="quotelev1">&gt; explain the fantastic outcomes of evolution, as opposed to any  
</em><br />
<em class="quotelev1">&gt; power inhering in the class of relatively simple Darwinian  
</em><br />
<em class="quotelev1">&gt; algorithms.  In the mutation-absorption model of the enzyme, we can  
</em><br />
<em class="quotelev1">&gt; begin to see how computational power emerges from the breakdown in  
</em><br />
<em class="quotelev1">&gt; the simulation relation, by the failure of commutativity (and, in  
</em><br />
<em class="quotelev1">&gt; the mathematical sense, the creation of torsion).  This seems to be  
</em><br />
<em class="quotelev1">&gt; the vexing locus of this new field: clarifying precisely what  
</em><br />
<em class="quotelev1">&gt; happens when formal systems fail to track changes in fragments of  
</em><br />
<em class="quotelev1">&gt; nature.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; In this FIS session on molecular bionetworks, I think we may have  
</em><br />
<em class="quotelev1">&gt; the opportunity to find a clearer understanding here.  Jerry  
</em><br />
<em class="quotelev1">&gt; Chandler's work deals with the connection between the formal (or  
</em><br />
<em class="quotelev1">&gt; symbolic) and the biological, and it seems these two perspectives  
</em><br />
<em class="quotelev1">&gt; may be mutually illuminating, and give us all the chance for some  
</em><br />
<em class="quotelev1">&gt; brainstorming in this largely unexplored area. Perhaps natural  
</em><br />
<em class="quotelev1">&gt; computing takes us back to the medieval meaning of natural as  
</em><br />
<em class="quotelev1">&gt; &quot;sublunar&quot; (to bring up term recirculated lately by Pedro  
</em><br />
<em class="quotelev1">&gt; Marijuan), dealing with the mutable substances down here under the  
</em><br />
<em class="quotelev1">&gt; orbit of the moon, illuminated by, but separated from, the  
</em><br />
<em class="quotelev1">&gt; immutable forms of the celestial world.
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; One final introductory thought. In imputing computing to a fragment  
</em><br />
<em class="quotelev1">&gt; of nature -- a ribosome, say -- we should not view ourselves as  
</em><br />
<em class="quotelev1">&gt; reducing it to the apparently impoverished level of a Turing  
</em><br />
<em class="quotelev1">&gt; machine or a Pentium 4. Computer Science is in its infancy; the  
</em><br />
<em class="quotelev1">&gt; ribosome, the cell, the ecosystem: these are all better exemplars  
</em><br />
<em class="quotelev1">&gt; of computational wonder than our humble devices. If only we had a  
</em><br />
<em class="quotelev1">&gt; theory…
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; ____________________________________
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt;
</em><br />
<em class="quotelev1">&gt; _______________________________________________
</em><br />
<em class="quotelev1">&gt; fis mailing list
</em><br />
<em class="quotelev1">&gt; fis&#64;listas.unizar.es
</em><br />
<em class="quotelev1">&gt; <a href="../../webmail.unizar.es/mailman/listinfo/fis">http://webmail.unizar.es/mailman/listinfo/fis</a>
</em><br />
<em class="quotelev1">&gt;
</em><br />
<p>Jerry LR Chandler
<br />
Research Professor
<br />
Krasnow Institute for Advanced Study
<br />
George Mason University
<br />
<span id="received"><dfn>Received on</dfn> Mon Nov 21 08:04:48 2005</span>
</div>
<!-- body="end" -->
<div class="foot">
<map id="navbarfoot" name="navbarfoot" title="Related messages">
<ul class="links">
<li><dfn>This message</dfn>: [ <a href="#start">Message body</a> ]</li>
<!-- lnext="start" -->
<li><dfn>Next message</dfn>: <a href="2236.html" title="Next message in the list">Loet Leydesdorff: "[Fis] Journal Citation Reports of _Science Citation Index_ and _SocialScience Citation Index_ combined"</a></li>
<li><dfn>Previous message</dfn>: <a href="2234.html" title="Previous message in the list">Aleks Jakulin: "Re: FW: [Fis] The Molecule as Text"</a></li>
<!-- lnextthread="start" -->
<li><dfn>Next in thread</dfn>: <a href="2250.html" title="Next message in this discussion thread">Jerry LR Chandler: "[Fis] Re: ON MOLECULAR BIONETWORKS (III)"</a></li>
<li><a name="replies" id="replies"></a><dfn>Reply</dfn>:  <a href="2250.html" title="Message sent in reply to this message">Jerry LR Chandler: "[Fis] Re: ON MOLECULAR BIONETWORKS (III)"</a></li>
<!-- lreply="end" -->
</ul>
<ul class="links">
<li><a name="options2" id="options2"></a><dfn>Contemporary messages sorted</dfn>: [ <a href="date.html#2235" title="Contemporary messages by date">By Date</a> ] [ <a href="index.html#2235" title="Contemporary discussion threads">By Thread</a> ] [ <a href="subject.html#2235" title="Contemporary messages by subject">By Subject</a> ] [ <a href="author.html#2235" title="Contemporary messages by author">By Author</a> ] [ <a href="attachment.html" title="Contemporary messages by attachment">By messages with attachments</a> ]</li>
</ul>
</map>
</div>
<!-- trailer="footer" -->
<p></td></table>

<br><p>
<SMALL> 
<EM> 
This archive was generated by  <A HREF="../../www.hypermail.org/default.htm">hypermail 2.1.8</A> on Mon 21 Nov 2005 - 08:04:48 CET
</EM> 
</SMALL> 

<script language="javascript1.2" src="../scripts/menu.js"></script>
</body></html>